docs/getting_started.md

```diff
@@ -96,7 +96,7 @@ The project Makefile provides a convenient make target to generate the yaml file
 ```bash
 # vCenter config/credentials
 export VSPHERE_SERVER=10.0.0.1                # (required) The vCenter server IP or UR
-export VSPHERE_USER=viadmin@vmware.local      # (required) The vCenter user to login with
+export VSPHERE_USERNAME=viadmin@vmware.local      # (required) The vCenter user to login with
```

Comment:
Hi @tkrausjr,

The alignment on line 99 is now incorrect. Please fix or let this PR be replaced by #412. Either way is fine.

Reply:
Done